helm/nginx-ingress-controller-app/templates/controller-deployment.yaml

```diff
@@ -28,7 +30,6 @@ spec:
         giantswarm.io/service-type: "managed"
         k8s-app: {{ .Values.controller.k8sAppLabel }}
       annotations:
-        releaseRevision: {{ $.Release.Revision | quote }}
```

Comment:
Yes this likely got copied from somewhere else since its a pattern we often use.

Since the ingress controller can do config reloads this will reduce the number of times pods are rolled. Nice catch. 👍